src/markdoc/tags/Tabs.svelte

```diff
@@ -26,7 +26,7 @@
 </script>
 
 <div class="aw-card is-normal u-margin-block-start-16" {...$root} use:root>
-	<nav class="tabs u-flex u-gap-16">
+	<nav class="tabs u-flex u-gap-16" aria-label="Tabs">
```

Comment:
I think we can remove the `nav` element here. Multiple tabs may be used in markdown, resulting in duplicated aria labels, which will be more frustrating then if these weren't navigation elements.